com.unity.render-pipelines.high-definition/Documentation~/Post-Processing-Depth-of-Field.md

```diff
@@ -19,6 +18,7 @@ Depth Of Field includes [more options](More-Options.html) that you must manually
 | **Property**       | **Description**                                              |
 | ------------------ | ------------------------------------------------------------ |
 | **Focus Mode**     | Use the drop-down to select the mode that HDRP uses to set the focus for the depth of field effect.<br />&#8226; **Off**: Select this option to disable depth of field.<br />&#8226; **Use Physical Camera**: Select this option to use the physical [Camera](HDRP-Camera.html) to set focusing properties for the depth of field effect. For information on what Camera properties affect depth of field, see [Physical Camera settings](#PhysicalCameraSettings).<br />&#8226; **Manual**: Select this option to use custom values to set the focus of the depth of field effect. |
+| **Quality** | Select one of the **Low**, **Medium** or **High** quality presets or **Custom** quality to override some or all the settings localy.|
```

Comment:
Specifies the quality level to use for the effect. You can select either the **Low**, **Medium**, or **High** quality presets or, if you want to override properties locally, select **Custom** quality and provide your own values.